desktop/main.js

```diff
@@ -179,7 +179,6 @@ const mainWindow = (() => {
             }
 
             if (ELECTRON_ENVIRONMENT.isDev()) {
-                require('dotenv').config();
                 if (process.env.USE_WEB_PROXY !== 'false') {
                     browserWindow.webContents.session.webRequest.onHeadersReceived(validDestinationFilters, (details, callback) => {
                         // eslint-disable-next-line no-param-reassign
```

Comment:
BTW @roryabraham I'm not sure why do we even try to use the web proxy 
1. Local (desktop) development works without it
2. We're not doing the same running ios/android locally

Oh and we're not even starting the proxy server e.g. see the `web` script:
https://github.com/Expensify/App/blob/a98df645420cdb9cdcdc7977bb3daffbe0c0c883/package.json#L17

Reply:
Looks like this isn't related to the proxy but the `origin` 
For DEV the origin is `localhost`, while for staging/prod it would be `app://-`. 
So on DEV we shouldn't use the proxy flag but always override requests
![image](https://user-images.githubusercontent.com/12156624/155032386-aedb18ea-d4fe-41f5-b07f-711210afc30f.png)

Looks like the origin depends on this block: 
https://github.com/Expensify/App/blob/6ef2e519d1ecf5b0e6db3f5498e24c7e50e9f21b/desktop/main.js#L127-L129

Using `win.loadURL` would use the origin of that url, while `serve` is serving from the app package and sets that as origin